package.json

```diff
@@ -21,7 +21,7 @@
     "latex"
   ],
   "jupyterlab": {
-    "extension": true
+    "extension": "lib/latex"
```

Comment:
What does that do?

Reply:
With the PDFJS extension we are including two different types of lab extensions: a normal one and a mimerenderer one. This is indicating the separate entry points for those two extensions.